clang-tools-extra/test/clang-tidy/checkers/bsl-identifier-typographically-unambiguous.cpp

```diff
@@ -0,0 +1,27 @@
+// RUN: %check_clang_tidy %s bsl-identifier-typographically-unambiguous %t
+
+// FIXME: Add something that triggers the check here.
+void frn0();
+void frno();
+void frn();
+void fm();
+void f___m();
+
+int id1_a_b_c;
```

Comment:
Does this test actually detect things that are not-compliant? This doesn't seem like this test is complete. I think there should be some LLVM specific comments that detect the non-compliant cases to prove that this code is working.